tests/Fixer/Phpdoc/NoBlankLinesAfterPhpdocFixerTest.php

```diff
@@ -160,6 +160,58 @@ public function testLineBeforeUseStatementIsNotRemoved(): void
         $this->doTest($expected);
     }
 
+    /**
+     * @dataProvider provideLineBeforeRequireCases
+     */
+    public function testLineBeforeRequireIsNotBeRemoved(string $expected, ?string $input = null): void
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public static function provideLineBeforeRequireCases(): iterable
```

Comment:
```suggestion
     * @dataProvider provideLineBeforeIncludeOrRequireIsNotRemovedCases
     */
    public function testLineBeforeIncludeOrRequireIsNotRemoved(string $expected, ?string $input = null): void
    {
        $this->doTest($expected, $input);
    }

    public static function provideLineBeforeIncludeOrRequireIsNotRemovedCases(): iterable
```

- test is about `require`, but also about `include`, so let's make it more precise
- "NotBeRemoved" is grammatically incorrect
- provider's name should reflect test itself (`php_unit_data_provider_name`)

Reply:
That "NotBeRemoved" is copied from `function testLineBeforeDeclareIsNotBeRemoved()` so that should change as well then.